lib/experimental/block-editor-settings-mobile.php

```diff
@@ -33,7 +71,71 @@ function gutenberg_get_block_editor_settings_mobile( $settings ) {
 		$settings['__experimentalEnableListBlockV2'] = true;
 	}
 
-	return $settings;
+	return keep_supported_block_editor_settings_mobile(
+		$settings,
+		array(
+			'alignWide'                        => true,
+			'allowedBlockTypes'                => true,
+			'allowedMimeTypes'                 => true,
+			'defaultEditorStyles'              => true,
+			'blockCategories'                  => true,
+			'isRTL'                            => true,
+			'imageDefaultSize'                 => true,
+			'imageDimensions'                  => true,
+			'imageEditing'                     => true,
+			'imageSizes'                       => true,
+			'maxUploadFileSize'                => true,
+			'__unstableGalleryWithImageBlocks' => true,
+			'disableCustomColors'              => true,
+			'disableCustomFontSizes'           => true,
+			'disableCustomGradients'           => true,
+			'disableLayoutStyles'              => true,
+			'enableCustomLineHeight'           => true,
+			'enableCustomSpacing'              => true,
+			'enableCustomUnits'                => true,
+			'colors'                           => true,
+			'fontSizes'                        => true,
+			'__experimentalStyles'             => array(
+				'elements'   => true,
+				'spacing'    => true,
+				'blocks'     => true,
+				'color'      => true,
+				'typography' => true,
+			),
+			'__experimentalFeatures'           => array(
+				'appearanceTools'               => true,
+				'useRootPaddingAwareAlignments' => true,
+				'border'                        => true,
+				'color'                         => true,
+				'shadow'                        => true,
+				'spacing'                       => true,
+				'typography'                    => array(
+					'dropCap'        => true,
+					'fontSizes'      => true,
+					'fontStyle'      => true,
+					'fontWeight'     => true,
+					'letterSpacing'  => true,
+					'textColumns'    => true,
+					'textDecoration' => true,
+					'textTransform'  => true,
+					'writingMode'    => true,
+				),
+				'blocks'                        => true,
+				'background'                    => true,
+				'dimensions'                    => true,
+				'position'                      => true,
+			),
+			'gradients'                        => true,
+			'disableCustomSpacingSizes'        => true,
+			'spacingSizes'                     => true,
+			'__unstableIsBlockBasedTheme'      => true,
+			'localAutosaveInterval'            => true,
+			'__experimentalDiscussionSettings' => true,
+			'__experimentalDashboardLink'      => true,
```

Comment:
I didn't find many references to `__experimentalDashboardLink`. It appears to be a hook to customize the back button in the full site editor, which I do not believe is relevant to the the mobile editor. My thought is that this could be excluded. WDYT?

Reply:
Resolved w/ https://github.com/WordPress/gutenberg/pull/56424/commits/9c19187a4394697fa3e676b44462a10101063e66